### PR TITLE
Readds combat droids and pre-vended protein bars to Sulaco

### PR DESCRIPTION
**Pull request by Derrick2Electricboogaloo**
## About The Pull Request
Corrects and oversight that is overdue.

## Why It's Good For The Game
I made a boo-boo, and forgot to re-add the combat droids to the sulaco. This corrects that, and allows the AI to have their toys back onboard.

## Changelog
:cl:
fix: Readded combat droids to Sulaco
fix: Readded prevended protein bars to vendors 
/:cl:

### CODE REVIEW

**Comment by Derrick2Electricboogaloo** [comment]
God dammit, WHY DID IT ADD THAT FILE. It was one file before.

**Comment by Derrick2Electricboogaloo** [comment]
Screw it I guess it is two factor now.

**Review by RipGrayson — CHANGES_REQUESTED** [review]
This nanomed is stuck inside the wall, please fix it
caves13

These cameras have improper rotation, please correct them
caves12
caves14
caves15

**Comment by Derrick2Electricboogaloo** [comment]
> 

That is interesting. In my previous local tests they were fine. What changed? I can fix it.

File: code/modules/clothing/suits/marine_armor.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 	var/list/armor_overlays
 	actions_types = list(/datum/action/item_action/toggle)
 	flags_armor_features = ARMOR_LAMP_OVERLAY
-	flags_item = SYNTH_RESTRICTED|IMPEDE_JETPACK
+	flags_item = IMPEDE_JETPACK
```
Comment (BraveMole) [inline code/modules/clothing/suits/marine_armor.dm:40]: What is this doing in this pr ? I dont get the relation between the two feature

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/marine_armor.dm (hunk 1 above)
Comment (BraveMole) [reply code/modules/clothing/suits/marine_armor.dm:40]: so fix it ?

**Review by RipGrayson — APPROVED** [review]
Mapping side of this PR looks fine

File: code/modules/clothing/suits/marine_armor.dm (hunk 1 above)
Comment (Derrick2Electricboogaloo) [reply code/modules/clothing/suits/marine_armor.dm:40]: I guess I could try removing the file.

**Comment by BraveMole** [comment]
derrick was mean

**Comment by Derrick2Electricboogaloo** [comment]
Ah yeah!

**Comment by Derrick2Electricboogaloo** [comment]
All resolved for now for the M3 harness fix. That is my next PR.

**Review by BraveMole — CHANGES_REQUESTED** [review]
fix title and pr body

**Comment by Derrick2Electricboogaloo** [comment]
> fix title and pr body

Done.

**Pull request merged**